tools/fuzzer/CMakeLists.txt

```diff
@@ -40,19 +40,33 @@ add_executable(sof-fuzzer
 	platform/hsw-host.c
 )
 
+add_executable(sof-afl-fuzzer
+	main.c
+	qemu-bridge.c
+	topology.c
+	platform/byt-host.c
+	platform/hsw-host.c
```

Comment:
whats the effort to add cavs-host.c here ?

Reply:
@ranj063 lets add that to our list of milestones to figure out the effort here

Reply:
@lgirdwood @cujomalainey in terms of effort I think the mail missing items is the IRQ handlers. We have added this as a deliverable for the next phase of the project